client/a8c-for-agencies/sections/team/hooks/use-member-list.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useMemo } from 'react';
+import useFetchActiveMembers from 'calypso/a8c-for-agencies/data/team/use-fetch-active-members';
+import useFetchMemberInvites from 'calypso/a8c-for-agencies/data/team/use-fetch-member-invite';
```

Comment:
Can we update the file name to be plural?

Reply:
Good catch! This has been addressed.